### DISCUSSION TITLE
Unable to change organization member visibility

### DISCUSSION THREAD

**Discussion by thunder-007** [community/community#19481] [Repositories] [answered]
![Screenshot (304)](https://user-images.githubusercontent.com/75431678/175812907-76a50a94-224f-40c8-8a1f-cf774dbc1d55.png)
Unable to change members visibility in organization (education teams)

**Comment by melvincwng** [comment]
@thunder-007, are you the owner of that particular GitHub Organization? If you are, continue reading on.

It seems like by default, your organization's visibility setting is set to 'Private'. To toggle the visibility of your organization to 'Public', please follow the steps in this page of [GitHub Docs](https://docs.github.com/en/enterprise-server@3.1/admin/user-management/managing-organizations-in-your-enterprise/configuring-visibility-for-organization-membership).

Otherwise, members of any organization have the option to toggle their own visibility in an organization as well. For informing your individual members on how to toggle on their organization visibility, you can refer to [this link](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership) over here.

I hope this reply has helped you. 

Appreciate it, and have a nice day 😄

**Comment by thunder-007** [comment]
Those docs are outdated I mean the UI is completely different now btw I'm the owner of the organisation

**Comment by melvincwng** [comment, marked as answer]
@thunder-007, thanks for informing me that you are the organization owner.

Can you follow the steps in this other [GitHub Docs link](https://docs.github.com/en/enterprise-server@3.2/admin/user-management/managing-organizations-in-your-enterprise/configuring-visibility-for-organization-membership)?

It should detail step by step instructions on how to set the default organization visibility to 'Public' and enforce the same default setting for all new members.

Hope this clarifies and resolves your issue.

Let me know if this answer is helpful. If it is, just 'Mark as answer'. Otherwise, ping me and we shall see what other possible solutions are there.

**Reply by melvincwng** [reply]
@thunder-007, I realized I may have added back the same link back here.

If all fails, my final suggestion would be to raise a support ticket about your organization to [GitHub Support](https://support.github.com/contact/account).

Kindly describe your issue to the support engineers, and allow them a few working days at least, to rectify and resolve your issue.

Hope this is helpful.

**Reply by thunder-007** [reply]
Just raised a support ticket tq for assistance.

**Reply by melvincwng** [reply]
@thunder-007, no problem, grateful I was able to assist you :)

Please mark this reply as answer, so that others know this issue has been addressed.

Thanks!
